integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -0,0 +1,143 @@
+# Default values for alluxio.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+image: alluxio/alluxio
+imageTag: 2.1.0-SNAPSHOT
+imagePullPolicy: Always
+
+# Set the tieredstore
+tieredstore:
+  levels:
+# How to use multiple layers
+# level 0 must be MEM, and it's better to point to /dev/shm
+#  - alias: MEM
+#    level: 0
+#    path: /dev/shm
+#    #type: emptyDir
+#    type: hostPath
+#    quota: 10GB
+#    high: 0.95
+#    low: 0.7
+#  - alias: SSD
+#    level: 1
+## The path is inside the container
+#    path: /alluxio-ssd
+## the volume type, it may be hostPath, emptyDir
+#    type: hostPath
+#    quota: 30GB
+#    high: 0.9
+#    low: 0.7
+
+# How to use single layer
+  - alias: MEM
+    mediumtype: MEM,SSD
+    level: 0
+    path: /dev/shm,/alluxio-ssd
+    type: hostPath
+    quota: 10GB,50GB
+    high: 0.95
+    low: 0.7
+
+
+
+user: 0
```

Comment:
add a comment for each section. @jiacheliu3 is going to change the default to `1000`

Reply:
In our next step we will try to run everything as non-root user `alluxio:alluxio` with `uid=1000`. The PR is not ready yet. I'll keep you updated when we make that change @cheyang .

Reply:
@cheyang We are switching to running Alluxio as non-root user `alluxio` with `uid:1000` in this [PR](https://github.com/Alluxio/alluxio/pull/9966)